config/sidebar.yml

```diff
@@ -1237,6 +1237,9 @@ articles:
 
           - title: Datadog Event Streams
             url: /logs/streams/datadog
+            
+          - title: Azure Event Grid Streams
+            url: /logs/streams/azure-event-grid
```

Comment:
Will we be adding this file in a subsequent PR?

Reply:
@Deborah-Digges it was a draft. I added the article now. Please review again 😄

Reply:
Sorry didn't realize it was a draft 😓